resource_locking/starter.py

```diff
@@ -0,0 +1,67 @@
+import asyncio
+from typing import Any
+
+from temporalio.client import Client, WorkflowFailureError, WorkflowHandle
+from temporalio.common import WorkflowIDConflictPolicy
+
+from resource_locking.lock_manager_workflow import (
+    LockManagerWorkflow,
+    LockManagerWorkflowInput,
+)
+from resource_locking.resource_locking_workflow import (
+    ResourceLockingWorkflow,
+    ResourceLockingWorkflowInput,
+)
+from resource_locking.shared import LOCK_MANAGER_WORKFLOW_ID
+
+
+async def main():
+    # Connect client
+    client = await Client.connect("localhost:7233")
+
+    # Start the ResourceLockingWorkflows
+    resource_locking_handles: list[WorkflowHandle[Any, Any]] = []
+    for i in range(0, 4):
+        input = ResourceLockingWorkflowInput(
+            iteration_to_fail_after=None,
+            should_continue_as_new=False,
+        )
+        if i == 0:
+            input.should_continue_as_new = True
+        if i == 1:
+            input.iteration_to_fail_after = "first"
+
+        resource_locking_handle = await client.start_workflow(
+            workflow=ResourceLockingWorkflow.run,
+            arg=input,
+            id=f"resource-locking-workflow-{i}",
+            task_queue="default",
+        )
+        resource_locking_handles.append(resource_locking_handle)
+
+    # Add some resources
+    lock_manager_handle = await client.start_workflow(
```

Comment:
Why (maybe) start your one global lock manager after all of its dependents? Why not first?

Reply:
It was to exercise signal-with-start in the dependents, but the need for that is gone now.